<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvScoreGradientWidget</class>
 <widget class="QWidget" name="vvScoreGradientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>209</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="qtGradientWidget" name="gradient" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QComboBox" name="interpolationMode">
     <item>
      <property name="text">
       <string>Discrete</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Smooth (Linear)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Smooth (Cubic)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>47</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="addStop">
     <property name="text">
      <string>Add Stop</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons/vvwidgets.qrc">
       <normaloff>:/icons/16x16/add</normaloff>:/icons/16x16/add</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="removeStop">
     <property name="text">
      <string>Remove Stop</string>
     </property>
     <property name="icon">
      <iconset resource="../../Icons/vvwidgets.qrc">
       <normaloff>:/icons/16x16/delete</normaloff>:/icons/16x16/delete</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="vvScoreGradientStopTree" name="stops">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qtGradientWidget</class>
   <extends>QWidget</extends>
   <header location="global">qtGradientWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>vvScoreGradientStopTree</class>
   <extends>QTreeWidget</extends>
   <header>vvScoreGradientStopTree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Icons/vvwidgets.qrc"/>
 </resources>
 <connections/>
</ui>
